python/lsst/meas/algorithms/readTextCatalogTask.py

```diff
@@ -54,6 +54,12 @@ class ReadTextCatalogConfig(pexConfig.Config):
         doc=("Format of files to read, from the astropy.table I/O list here:"
              "http://docs.astropy.org/en/stable/io/unified.html#built-in-table-readers-writers")
     )
+    fill_values = pexConfig.ListField(
+        dtype=str,
+        default=[],
```

Comment:
Default `[]` or `None`?

Reply:
I changed this to `None` and added an `optional=True` so it passes validation.